data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+			`	The voice continues: "I know what you seek here. The trinket you covet is of little relevance, even for your own, human standards. But if you wish to possess it, then take it. Perhaps the sight of that vessel will remind the living races of those who have come and gone... and how, despite their great accomplishments, their decision to make war reduced them to graveyards, relics, and memories."`
+			`	The voice, and the overwhelming emotions that came alongside it, disappear as mysteriously as they came. Desyo, who witnessed you speaking into thin air, asks whether you feel okay. It seems like no one else heard it.`
+
+	npc board
+		government "Sheragi"
+		personality derelict mute
+		system Zubenelhakrabi
+		ship "Emerald Sword" "Emerald Sword"
+		conversation
+			`You dock with the Emerald Sword, and the crew starts working to carefully find a way in, directed by the Hai archaeologists. There is no atmosphere inside it as the life support has long since ceased to work, and it takes a long time to restore it through your ship's own life support and patch tiny micrometeorite holes. Once it's in breathable conditions, it still smells like a tomb. You find many Sheragi skeletons and mummified remains. A few of them have prosthetics that survived better than their host. Your crew gets to work cleaning up the thick layer of dust and grime that covers every surface, while the more specialized crew tries to bring the power core back to life and take advantage of the zero-G environment to move the jump drive into the ship more easily. You'll need to land to properly install it, however. All that is left for you to do is relocate the many corpses littering the living spaces with your crew.`
+	
+	on complete
+		conversation
+			`It takes some time to find a suitable landing spot - high enough to escape the highest concentrations of chlorine gas in the atmosphere but flat enough to land on. You eventually park your ships over a mesa. The environmental suits brought along "just in case" turn out to be very useful here. The atmosphere would be otherwise unbreathable for long periods of time.`
+			`	Over the course of the next day, the workers unload some outfitting and repairing equipment, while scientists slowly work out the functions of the ship, and brief you and your crew on how to operate it. One of the workers invite you to visit the temporary outfitter and oversee the proper installation of the Outfits Expansion and the Jump Drive in the Emerald Sword.`
+			`[You need to remove the Damaged Sheragi Systems to repair it.]`
+
+mission "Emerald Sword Epilogue"
+	name "The Emerald Sword"
+	description "Return to Zug"
+	landing
+	source "Echoes of Tragedy"
+	destination "Zug"
+	passengers 20
+	cargo "Outfitting Equipment" 25
+	
+	to offer
+		has "The Emerald Sword 3: done"
+	
+	blocked `You need more bunks or cargo.`
```

Comment:
I believe it's (nearly) impossible for this mission to be blocked given that the previous mission had higher requirements, so this blocked dialog isn't necessary.

Reply:
Big oops, i left that as a debug message because that mission kept failing to get offered when i was making it.